README.md

```diff
@@ -1,8 +1,215 @@
+
+# 📖 Yacht Rental <a name="about-project"></a>
+
+**Yacht Rental** is the react client of  a full stack yacht reservation web application built on the rails API.The main function of the app is to allow users to book reservations for renting a yacht.
+
+=> [Here is the Yacht Rental backend-API](https://github.com/K0ppai/yacht-rental-backend.git)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+ <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">ReactJS</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://tailwindcss.com/">Tailwind CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **The user can signup and login to the website to make reservation.**
+- **The user can see links to: Yachts, Reserve, My Reservations, Add Yacht and Delete Yacht**
+- **The user can see a list of yachts**
+- **The user  can select the item and see details of the item**
+- **The user clicks the "Add yacht" link in the navigation panel they can see a form for adding a new yacht.**
+- **When the user clicks the "Delete yacht" link in the navigation panel they can see a list of all items with title and "Delete" button.**
+- **When the user clicks the "My reservations" link in the navigation panel they can see a list of their reservations (with information about item name, date and city).**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="#">Yacht Rental Live</a>
+- <a href="#">Yacht Rental Backend API</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Kanban Board <a name="Kanban-Board"></a>
+
+ ## A Team of 3 Members worked on this project - [See list of authors](#authors)
+   - <a href="https://github.com/K0ppai/yacht-rental-backend/projects/1">Kanban Board Link</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- `React`
+- `Redux`
+- `Rails`
+- `Tailwind CSS and HTML`
+- `Code editor (VSCode)`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/HtetWaiYan7191/yacht_rental_front_end.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd yacht_rental_front_end
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run dev or yarn run dev
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
```

Comment:
- [ ] This command is not going to work because the script does not exist in `package.json`:

![image](https://github.com/HtetWaiYan7191/yacht_rental_front_end/assets/57421823/268e1d61-2d4f-4a91-9d30-3e560754d6bb)

Also, I failed to find any frontend tests.

Kindly, either remove the section or add some tests.

Also, in the case of tests please add the script to `package.json` so the command works. 👍